src/components/select/Select.js

```diff
@@ -183,11 +183,20 @@ class Select extends PureComponent {
     };
   };
 
-  getMenuStyles = (base) => ({
-    ...base,
-    marginTop: 3,
-    marginBottom: 3,
-  });
+  getMenuStyles = (base) => {
+    const { menuVerticalOffset } = this.props;
+    return {
+      ...base,
+      marginTop: 3,
+      marginBottom: 3,
+      ...(menuVerticalOffset
```

Comment:
Isn't this horizontal instead of vertical?

Reply:
💩 you're right